spec/requests/reservations_tickets_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'ReservationsTickets', type: :request do
+  let(:movie) { create(:movie) }
+  let(:hall) { create(:hall) }
+  let(:seance) { create(:seance, hall_id: hall.id, movie_id: movie.id) }
+  let(:reservation) { create(:reservation, seance_id: seance.id) }
+
+  before do
+    movie
+    hall
+    seance
+    reservation
```

Comment:
Consider that `reservation` creates `seance` and `seance` creates `movie` and `hall` so it's enough to put `reservation` in before block cause this includes `seance, movie, hall`

Reply:
I am not sure how - do I need to add that to each factory? 🤔

Reply:
No, no. You did everything well! It might be simplified by changes in factory but it's not mine point here :)

I just wanted to say that you can just put here:

``` before { reservation }```

And this one line will do exactly the same like here:
```
  before do
    movie
    hall
    seance
    reservation
  end
```
cause reservation create includes creation of other elements :)

Reply:
Aaaah! I get it now! You are right, thank you! :)